airflow-core/src/airflow/observability/stats.py

```diff
@@ -22,11 +22,11 @@
 from collections.abc import Callable
```

Comment:
Since this file is moved from airflow.stats to airflow.observability.stats, we must add a compatibility shim / deprecation warning for people trying to import from old path. Check how it done in here and do it similarly: https://github.com/apache/airflow/blob/main/airflow-core/src/airflow/__init__.py